dist/tools/packer/preseed.cfg

```diff
@@ -26,7 +26,7 @@ d-i pkgsel/include string openssh-server ntp curl nfs-common build-essential \
     python-setuptools libusb-1.0-0 libusb-1.0-0-dev libudev-dev libftdi1 \
     libftdi-dev libftdipp1-dev libftdipp1-2v5 libhidapi-dev libhidapi-hidraw0 \
     libhidapi-libusb0 cmake autotools-dev autoconf pkg-config jimsh libtool \
-    valgrind openocd python-serial python3-serial
+    valgrind openocd python-serial python3-serial xsltproc
```

Comment:
This has nothing to do with importing Keccak. Please provide a separate PR for this new package.

Reply:
Actually, I forgot to mention this in the PR description but it is necessary if you want to build the Keccak package in a Vagrant VM. Should I still provide a separate PR for this?

Reply:
Yes please. People that maintain the vagrant image are different people than the people reviewing security stuff.

Reply:
I've removed these changes from the branch for this PR.